release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2730,6 +2730,45 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, during node reboots, especially during upgrade operations, the node that interacts with the rebooting machine enters a `Ready=Unknown` state for a short amount of time. This situation caused the Control Plane Machine Set Operator to enter an `UnavailableReplicas` condition and then an `Available=false` state. The `Available=false` state triggers alerts that demand urgent action, but intervention would only be required for a short period until the node reboots. With this release, a grace period for node unreadiness is provided where if a node enters an unready state, the Control Plane Machine Set Operator will not instantly enter an `UnavailableReplicas` condition or an `Available=false` state. (link:https://issues.redhat.com/browse/OCPBUGS-34971[*OCPBUGS-34971*]).
+
+* Previously, the OpenShift Cluster Manager container did not have the right TLS Certificates. As a result, image streams could not be used in disconnected deployments. With this update, the TLS Certificates are added as projected volumes. (link:https://issues.redhat.com/browse/OCPBUGS-34580[*OCPBUGS-34580*])
+
+* Previously, when a serverless function was created in the create serverless form, BuilldConfig was not created. With this update, if the Pipelines Operator is not installed, if the pipeline resource is not created for particular resource, or if the pipeline is not added while creating a serverless function, it will create BuildConfig as expected. (link:https://issues.redhat.com/browse/OCPBUGS-34350[*OCPBUGS-34350*])
+
+* Previously, the reduction of the network queue did not work as expected for inverted rules such as `!ens0`. This happened because the exclamation mark symbol was duplicated in the generated tuned profile. With this release, the duplication no longer occurs so that inverted rules would as intended. (link:https://issues.redhat.com/browse/OCPBUGS-33929[*OCPBUGS-33929*]).
+
+* Previously, registry overrides applied to non-relevant data plane components. With this release, registry overrides no longer apply to these components. (link:https://issues.redhat.com/browse/OCPBUGS-33627[*OCPBUGS-33627*]).
+
+* Previously, when installing a cluster on {vmw-first}, the installation would fail if an ESXi host was in maintenance mode due to the installation program failing to retrieve version information from the host. With this update, the installation program does not attempt to retrieve version information from ESXi hosts that are in maintenance mode, allowing the installation to proceed. (link:https://issues.redhat.com/browse/OCPBUGS-31387[*OCPBUGS-31387*])
```

Comment:
```suggestion
* Previously, when installing a cluster on {vmw-first}, the installation failed if an ESXi host was in maintenance mode because the installation program could not retrieve version information from the host. With this update, the installation program does not attempt to retrieve version information from ESXi hosts that are in maintenance mode, allowing the installation to proceed. (link:https://issues.redhat.com/browse/OCPBUGS-31387[*OCPBUGS-31387*])
```

Reply:
:heavy_check_mark: